### PR TITLE
feat: new repo structure to support kubernetes resources

### PR DESCRIPTION
**Pull request by davelanglois-ssc**
The /*/configcontroller folder is where GCP resources are defined using their config connector schema.
The /*/kubernetes/<fleet-id>/**/<namespace> folder is where resources that have to be provisionned in a kubernetes namespace are defined. The fleet-id is the GCP project-id where the kubernetes clusters are deployed.

Attention: removing tagging solution for now until a fix is developped to support all use cases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,33 @@
...
-Tags are generated only when changes are affecting the `/tier2` folder. This functionality is possible because of the `version-tagging` pipeline.
-
-The tags will be created by that pipeline job after commits are merge to the `main` branch.
+## Branch Protection
 
-### **IMPORTANT**
+The main branch of this repository is protected meaning that pushing new commit to it will be denied. To implement changes, A Pull Request has to be completed.
 
-We recommend using those tags in the `setters-version.yaml` under the `/csync/source-customization` folder to define what has to be observed by the configsync operator.
+Every other branches configured to be observed by ConfigSync will also have a a branch protection rule defined.
```
Comment (lucstjean-ssc) [inline README.md:34]: remove the extra a in `also have a a branch`

File: README.md [hunk 2]
```diff
@@ -2,32 +2,33 @@
...
 
-## Tags
-
-Tags are generated only when changes are affecting the `/tier2` folder. This functionality is possible because of the `version-tagging` pipeline.
-
-The tags will be created by that pipeline job after commits are merge to the `main` branch.
+## Branch Protection
 
-### **IMPORTANT**
+The main branch of this repository is protected meaning that pushing new commit to it will be denied. To implement changes, A Pull Request has to be completed.
```
Comment (lucstjean-ssc) [inline README.md:32]: I would change to either pushing a new commit or pushing new commits

File: csync/tier2/configcontroller/source-base/root-sync-git/Kptfile [hunk 3]
```diff
@@ -7,7 +7,7 @@ metadata:
 info:
   description: |
     Landing Zone v2 package
-    A package to setup a git repository root sync
+    A package to setup a t2 git repository root sync
```
Comment (lucstjean-ssc) [inline csync/tier2/configcontroller/source-base/root-sync-git/Kptfile:10]: Can we change t2 to tier2?

Comment (lucstjean-ssc) [reply csync/tier2/configcontroller/source-base/root-sync-git/Kptfile:10]: I see what you did there. Please disregard this comment.

**Review by stephanemillaire-ssc — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (davelanglois-ssc) [reply README.md:34]: addressed with latest commit

File: README.md (hunk 2 above)
Comment (davelanglois-ssc) [reply README.md:32]: addressed with latest commit

**Review by anoopsidhu-ssc — APPROVED** [review]
reviewed the changes looks good

**Review by stephanemillaire-ssc — APPROVED** [review]

**Pull request merged**